docs/utils/proxyMap.mdx

```diff
@@ -1,7 +1,9 @@
 ---
-title: 'addComputed'
+title: 'proxyMap'
 section: 'Utils'
 description: ''
 ---
 
-# addComputed
+# proxyMap
+
+TODO
```

Comment:
@fkhadra Feel free to work on `proxyMap` and `proxySet`. Nobody else will be working on them.